openedx/features/assessment/tests/test_helpers.py

```diff
@@ -0,0 +1,247 @@
+        """ Test auto score without any ora submissions """
+
+        chapter1 = self.course_chapters[0].children[0]
+        chapter1_sequential1 = modulestore().get_item(chapter1).children[0]
+        chapter1_sequential1_vertical1 = modulestore().get_item(chapter1_sequential1).children[0]
+        chapter1_sequential1_vertical1_block1 = modulestore().get_item(chapter1_sequential1_vertical1).children[0]
+
+        anonymous_user = AnonymousUserIdFactory.create(user=self.user, course_id=self.source_course.id)
+
+        student = {
+            'anonymous_user_id': anonymous_user.anonymous_user_id
+        }
+        helpers.autoscore_ora(self.source_course.id, unicode(chapter1_sequential1_vertical1_block1), student)
+
+        assert mock_log_info.not_called
+        assert mock_log_war.called_once
+        assert mock_reset_score.not_called
+        assert mock_set_score.not_called
+
+    def test_select_options_with_two_point_rubric(self):
+        """ Validate rubric points calculation with two points rubric """
+
+        expected_options_selected = {'Ideas': 'Fair', 'Content': 'Fair'}
+        two_points_rubric_expected_earned_points = 4
+        two_points_rubric_expected_possible_points = 4
+
+        two_point_rubric_options_selected, two_point_rubric_earned_points, two_point_rubric_possible_points = helpers.select_options(
+            TWO_POINT_RUBRIC_DICTIONARY)
+
+        self.assertEqual(expected_options_selected, two_point_rubric_options_selected)
+        self.assertEqual(two_point_rubric_earned_points, two_points_rubric_expected_earned_points)
+        self.assertEqual(two_point_rubric_possible_points, two_points_rubric_expected_possible_points)
+
+    def test_select_options_with_three_point_rubric(self):
+        """ Validate rubric points calculation with two points rubric """
+
+        expected_options_selected = {'Ideas': 'Fair', 'Content': 'Fair'}
+
+        three_points_rubric_expected_earned_points = 4
+        three_points_rubric_expected_possible_points = 6
+
+        three_point_rubric_options_selected, three_point_rubric_earned_points, three_point_rubric_possible_points = helpers.select_options(
+            THREE_POINT_RUBRIC_DICTIONARY)
+
+        self.assertEqual(expected_options_selected, three_point_rubric_options_selected)
+        self.assertEqual(three_point_rubric_earned_points, three_points_rubric_expected_earned_points)
+        self.assertEqual(three_point_rubric_possible_points, three_points_rubric_expected_possible_points)
+
+    def test_get_philu_bot(self):
+        """ Verify that bot user is created if not already exists """
+        helpers.get_philu_bot()
+        self.assertEqual(True, User.objects.filter(username=PHILU_BOT_NAME).exists())
+
+    def test_get_rubric_for_course(self):
+        """ Test rubric for course """
+
+        chapter1 = self.course_chapters[0].children[0]
+        chapter1_sequential1 = modulestore().get_item(chapter1).children[0]
```

Comment:
Please rewrite this test by mocking return value of `prompts` and `rubric_criteria`.

Mock `get_item`, mocked object can return your `prompts` and `rubric_criteria`.

Assert mock_get_item called with exact arguments and return value